src/display/api.js

```diff
@@ -2003,6 +2003,7 @@ class WorkerTransport {
           });
           break;
         case 'FontPath':
+        case 'FontType3Res':
```

Comment:
I may be missing something, but why can't we just stick these in the `objs` so we don't have to do all the typ3e checks in canvas?

Reply:
That's what's currently happening without this patch, and that breaks because fonts are common to the *entire* document (i.e. placed in `commonObjs`), whereas the `objs` are unique for *each* page.
Since the same font is used on multiple pages, but the image resources were only available to the first page, using `objs` cannot work for these kind of font resource.

Hence the only simple way, that I could find at least, was to make these resources available "globally" which thus necessitated small changes in `src/display/canvas.js`.